setup.py

```diff
@@ -2,8 +2,8 @@
 from setuptools import setup
 
 MAJOR_VERSION = "0"
-MINOR_VERSION = "10"
-MICRO_VERSION = "95"
+MINOR_VERSION = "11"
+MICRO_VERSION = "0"
 VERSION = "{}.{}.{}".format(MAJOR_VERSION, MINOR_VERSION, MICRO_VERSION)
```

Comment:
can leave it at minor=10 as from the perspective of cliche there is no significant/breaking change yet